app.js

```diff
@@ -211,3 +211,19 @@ var employeeTableBody = document.getElementById('employee-table-body');
 var tableBody;
 
 printTable();
+
+
+var formEl = document.getElementById('form');
+formEl.addEventListener('submit', handleSubmit);
+function handleSubmit(event) {
+  event.preventDefault();
+  console.log(event.target.store.value);
+  var store = event.target.store.value;
+  var min = event.target.min.value;
```

Comment:
Keep in mind that these values come from `.value` in the form of a string, you probably want to run `parseInt` on the data coming in from the input.